crypt4gh/keys/__init__.py

```diff
@@ -155,6 +157,30 @@ def run(argv=sys.argv[1:]):
 
     pubkey = os.path.expanduser(args['--pk'])
     seckey = os.path.expanduser(args['--sk'])
+    comment = args['-C'].encode() if args['-C'] else None
+
+    if args['--relock']:
+        seckeypath = os.path.expanduser(seckey)
+        if not os.path.exists(seckeypath):
+            raise ValueError('Secret key not found')
+        old_passphrase = os.getenv('C4GH_PASSPHRASE')
+        cb = partial(getpass, prompt=f'Passphrase for {seckey}: ') if old_passphrase is None else lambda: old_passphrase
```

Comment:
The `partial` has to come first here or the whole statement is swallowed by the `lambda:` if it comes first.